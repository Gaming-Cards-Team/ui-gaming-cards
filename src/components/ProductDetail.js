import React from "react";
import Row from "react-bootstrap/Row";
import Col from "react-bootstrap/Col";

const imageStyle = {
  textAlign: "right",
};
const detailStyle = {
  color: "#2d963f",
  fontWeight: "bold",
};

const ProductDetail = () => (
  <div>
    <Row>
      <Col xs={5} style={imageStyle}>
        
      </Col>
      <Col xs={7} style={detailStyle}>
        
      </Col>
    </Row>
  </div>
);

export default ProductDetail;
