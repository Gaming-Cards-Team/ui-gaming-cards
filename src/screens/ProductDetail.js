import React from "react";
import Header from "../components/Header";
import ProductDetailBody from "../components/ProductDetailBody";
import Footer from "../components/Footer";

const ProductDetail = () => {
  return (
    <>
      <Header />
      <ProductDetailBody />
      <Footer />
    </>
  );
};

export default ProductDetail;
