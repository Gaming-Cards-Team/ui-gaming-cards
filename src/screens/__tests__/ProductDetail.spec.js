import React from 'react';
import renderer from 'react-test-renderer';
import ProductDetail from '../ProductDetail'; 

describe('Test ProductDetail component', () => {

    it('should validate the component', () => {

        const productDetail  = renderer.create(<ProductDetail />)
        expect(productDetail).toMatchSnapshot()
    });
});