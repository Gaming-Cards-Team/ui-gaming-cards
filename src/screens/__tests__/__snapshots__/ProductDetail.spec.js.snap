// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ProductDetail component should validate the component 1`] = `
Array [
  <nav
    className="navbar navbar-expand navbar-light bg-light fixed-top"
  >
    <a
      className="navbar-brand"
      href="#home"
    >
      <img
        alt="Level Up Logo"
        data-testid="logo"
        height="48"
        src="logo.png"
      />
    </a>
  </nav>,
  <div>
    <br />
    <br />
    <br />
    <div
      className="row"
    >
      <div
        className="col-lg-2 col-md-1 col-sm-12 col-12"
      />
      <div
        className="col-lg-3 col-md-4 col-sm-12 col-12"
        style={
          Object {
            "padding": "40px 0px",
            "textAlign": "center",
          }
        }
      >
        <img
          alt="Card"
          src="https://swdestinydb.com/bundles/cards/en/12/12081.jpg"
        />
      </div>
      <div
        className="col-lg-4 col-md-4 col-sm-12 col-12"
      >
        <div
          style={
            Object {
              "padding": "40px 20px",
            }
          }
        >
          <div>
            <span
              style={
                Object {
                  "color": "#2d963f",
                  "fontSize": "xx-large",
                  "fontWeight": "bold",
                }
              }
            >
               Destroy the Death Start
            </span>
            <h4>
              Covert Missions
            </h4>
            <div
              style={
                Object {
                  "paddingBottom": 15,
                  "paddingTop": 10,
                }
              }
            >
              <div
                className="card"
              >
                <div
                  className="card-body"
                >
                  <div
                    className="row"
                  >
                    <div
                      className="col-6"
                      style={
                        Object {
                          "borderRight": "1px solid #a09797e6",
                        }
                      }
                    >
                      <p>
                        <span
                          style={
                            Object {
                              "fontWeight": "bold",
                            }
                          }
                        >
                          Rarity: 
                        </span>
                        Uncommon
                      </p>
                      <p>
                        <span
                          style={
                            Object {
                              "fontWeight": "bold",
                            }
                          }
                        >
                          Card#: 
                        </span>
                        81
                      </p>
                      <p>
                        <span
                          style={
                            Object {
                              "fontWeight": "bold",
                            }
                          }
                        >
                          Card type: 
                        </span>
                        Plot
                      </p>
                      <p>
                        <span
                          style={
                            Object {
                              "fontWeight": "bold",
                            }
                          }
                        >
                          Affiliation: 
                        </span>
                        Hero
                      </p>
                      <p>
                        <span
                          style={
                            Object {
                              "fontWeight": "bold",
                            }
                          }
                        >
                          Color: 
                        </span>
                        Red
                      </p>
                      <p>
                        <span
                          style={
                            Object {
                              "fontWeight": "bold",
                            }
                          }
                        >
                          Point value(s): 
                        </span>
                        2
                      </p>
                      <p>
                        <span
                          style={
                            Object {
                              "fontWeight": "bold",
                            }
                          }
                        >
                          Uniqueness: 
                        </span>
                        Non/Unique
                      </p>
                    </div>
                    <div
                      className="col-6"
                    >
                      <p
                        style={
                          Object {
                            "fontWeight": "bold",
                          }
                        }
                      >
                        Description: 
                      </p>
                      After setup, give this plot 20 shields. You can resolve your 
                      <i>
                        pilot
                      </i>
                       and 
                      <i>
                        vehicle
                      </i>
                       dice showing ranged damage against this plot. 
                      <b>
                        Action
                      </b>
                       - If this plot has no shields, spend 1 resource to reroll 1 of your 
                      <i>
                        pilot
                      </i>
                       dice and 1 of your 
                      <i>
                        vehicle
                      </i>
                       dice. Then, if both of those dice are showing damage, you win the game.
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-lg-3 col-md-3 col-sm-12 col-12"
      />
    </div>
  </div>,
  <nav
    className="navbar navbar-expand navbar-light fixed-bottom"
    style={
      Object {
        "backgroundColor": "#2d963f",
      }
    }
  >
    <div
      data-testid="footer-text"
      style={
        Object {
          "color": "white",
          "textAlign": "center",
          "width": "100%",
        }
      }
    >
      Copyright Star Wars and all associated names and/or images are copyright Lucasfilm Ltd.
    </div>
  </nav>,
]
`;
